packages/tabs-react/src/Tabs.tsx

```diff
@@ -0,0 +1,187 @@
+    index: number;
+    setIndex: React.Dispatch<React.SetStateAction<number>>;
+    tabIDs: string[];
+    tabPanelIDs: string[];
+}
+
+/**
+ * Konteiner for elementer av typen Tab. Ikke funksjonell utenfor et Tabs element.
+ *
+ * Docs: https://jokul.fremtind.no/komponenter/tabs
+ */
+export const TabList = ({ children, ...injected }: TabListProps) => {
+    // props injected by Tabs
+    const { index, setIndex, tabIDs, tabPanelIDs, ...props } = injected as TabListProps & TabListInjectedProps;
+
+    const [tabsRect, setTabsRect] = useState<DOMRect>();
+    const [activeRect, setActiveRect] = useState<DOMRect>();
+
+    const tabsRef = useRef<HTMLDivElement>(null);
+    const activeRef = useRef<HTMLButtonElement>(null);
+
+    useLayoutEffect(() => {
+        setTabsRect(tabsRef.current?.getBoundingClientRect());
+        setActiveRect(activeRef.current?.getBoundingClientRect());
+    }, [index]);
+
+    const keyDownHandler = useCallback((event: React.KeyboardEvent<HTMLButtonElement>) => {
+        if (event.key === "ArrowLeft") {
+            const current = event.currentTarget;
+            const prev = event.currentTarget.previousSibling;
+
+            if (prev) {
+                (prev as HTMLElement).focus();
+            } else {
+                (current.parentElement?.lastChild?.previousSibling as HTMLElement).focus();
+            }
+        }
+
+        if (event.key === "ArrowRight") {
+            const current = event.currentTarget;
+            const next = event.currentTarget.nextSibling;
+
+            // skip the focus indicator element
+            if (next && next.nodeName === "BUTTON") {
+                (next as HTMLElement).focus();
+            } else {
+                (current.parentElement?.firstChild as HTMLElement).focus();
+            }
+        }
+    }, []);
+
+    return (
+        <div role="tablist" className="jkl-tablist" ref={tabsRef} {...props}>
+            {React.Children.map(children, (tab, tabIndex) => {
+                const isActive = index === tabIndex;
+
+                return React.isValidElement(tab)
+                    ? React.cloneElement(tab, {
+                          onKeyDown: keyDownHandler,
+                          tabIndex: isActive ? undefined : -1,
+                          ref: isActive ? activeRef : undefined,
+                          onClick: () => setIndex(tabIndex),
+                          "aria-selected": isActive,
+                          "aria-controls": tabPanelIDs[tabIndex],
+                          id: tabIDs[tabIndex],
+                      })
+                    : null;
+            })}
+
+            <span
+                className="jkl-tab__indicator"
+                style={{
+                    left: (activeRect?.left || 0) - (tabsRect?.left || 0),
+                    bottom: -1,
+                    width: activeRect?.width,
+                }}
+            />
+        </div>
+    );
+};
+
+interface Props {
+    children: ReactNode;
```

Comment:
Denne burde også kunne ta imot `className`.
```suggestion
    children: ReactNode;
    className?: string;
```
Jeg synes kanskje også den kan få `jkl-tabs` som standardklasse selv om rotelementet ikke har stil nå. Det vil gjøre det enklere å jobbe med fremover, og gjør det lettere å debugge den produserte HTML-en 😊 Tipper noen testere kanskje også ønsker seg mulighet for testid.